.github/workflows/ci.yml

```diff
@@ -0,0 +1,350 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - main
+
+jobs:
+
+  lint-build:
```

Comment:
you can remove all of the following CI jobs:

- test-codegen-build
- test-minimal-import-build
- test-pyinstaller-build
- release-builds
- qemu-release-builds
- sdist-build
- publish